pkg/util/util.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"context"
+	"github.com/blang/semver"
+	"github.com/google/go-github/github"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+const (
+	ghorg            = "redhat-developer"
```

Comment:
please add comments explaining what those variables are and what they are used for

Reply:
done